packages/main/src/popup-utils/PopoverRegistry.js

```diff
@@ -108,4 +127,21 @@ const getRegistry = () => {
 	return openedRegistry;
 };
 
+const getParentPopoversIfNested = instance => {
+	let currentElement = instance.parentNode;
+	const parentPopovers = [];
+
+	while (currentElement.parentNode) {
+		for (let i = 0; i < openedRegistry.length; i++) {
+			if (currentElement._id && currentElement._id === openedRegistry[i].instance._id) {
```

Comment:
We're trying to remove ids in a separate change, can you only compare the references without using the _ids here?